Example/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "react-native-reanimated": ["./react-native-reanimated.d.ts"]
+    }
+  },
+  "include": [
+    "index.js",
```

Comment:
is there no semantics like `"index.{js,ts,tsx}"` available?

Reply:
Not sure, but looks like we can just use `src/**/*`:

https://www.typescriptlang.org/tsconfig#include
> If a glob pattern doesn’t include a file extension, then only files with supported extensions are included (e.g. .ts, .tsx, and .d.ts by default, with .js and .jsx if allowJs is set to true).

Reply:
Fixed in 192e867e.